### ISSUE TITLE
[Feature Request]: Provide the ability to configure push notifications in Grafana OnCall

### ISSUE THREAD

**Issue by proffalken** [grafana/terraform-provider-grafana#1665]
### Feature Request

At present you can configure Mobile Push Notifications for Webhooks within the Grafana OnCall UI but not via Terraform.

The [API documentation](https://grafana.com/docs/oncall/latest/oncall-api-reference/integrations/#create-an-integration) is unclear as to which integrations you can configure and how to pass the appropriate values.

We would like to be able to add push notifications via Terraform so we can include them alongside all our other notification configuration that is already in Terraform.

![image](https://github.com/grafana/terraform-provider-grafana/assets/262545/3504b152-1481-49a8-838f-d2099f1dfbef)

**Comment by jsnod** [comment]
Yep, we're currently doing this until support for mobile push templates is added:

```hcl
resource "grafana_oncall_integration" "this" {
  ...

  templates {
    ...

    # Not yet supported by provider, but hopefully soon: https://github.com/grafana/terraform-provider-grafana/issues/1665
    # In the meantime, manually copy the below variables and paste into the Grafana template editor.
    # mobile_push {
    #   title   = var.mobile_push_title_template
    #   message = var.mobile_push_message_template
    # }
  }
}
```

**Comment by joeyorlando** [comment]
@proffalken this has been released in v3.8.2, please update your provider to give it a try 🙂
